resources/public/index.html

```diff
@@ -143,10 +143,10 @@
   if (hasLocalStorage()) {
     initMatomo();
     initTagManager();
-    
+
     if (!inIframe() && hasSessionStorage()) {
-      loadJS('/js/klaro/config.js', undefined, document.body);
-      loadJS('/node_modules/klaro/dist/klaro-no-css.js', undefined, document.body);
+      loadJS('https://s3.schnaq.com/schnaq-com/js/klaro/config.js', undefined, document.body);
```

Comment:
Ich bin ehrlich gesagt nicht happy damit das nun in einen separaten Bucket zu werfen 😅 wir haben den `schnaq-common`-Bucket, aus dem wir auch aus anderen Apps heraus die Assets lesen, nicht nur aus app.schnaq.com. Bitte pack die Config auch da mit rein, denn das ist schon der geteilte Bucket für mehrere Services.

Reply:
Können da aber auch gerne noch drüber reden :-) Mein Gedanke ist nur: wir haben schon einen geteilten Bucket, lass den weiter nutzen. Deine Gedanken kenne ich noch nicht.